### PR TITLE
[IPCT1-654] - Log of changes

### PR DESCRIPTION
**Pull request by dev-jotape**
This PR fixes [IPCT1-654] at https://impactmarket.atlassian.net/browse/IPCT1-654

## Changes
register a log of changes, when:
- when a community is edited
- when a user edits a profile

## Tests
new tests added in user.test file

### CODE REVIEW

File: packages/core/src/services/app/user.ts [hunk 1]
```diff
@@ -554,6 +556,13 @@ export default class UserService {
         if (updated[0] === 0) {
             throw new BaseError('UPDATE_FAILED', 'user was not updated!');
         }
+
+        LogService.saveLog(
+            user.address,
```
Comment (obernardovieira) [inline packages/core/src/services/app/user.ts:561]: you could use updated[0].id I think. THis goes together with my comment on "why not accepting just id?"

File: packages/core/src/services/app/appLog.ts [hunk 2]
```diff
@@ -0,0 +1,109 @@
...
+    community: {
+        id: number,
+        name: string,
+    },
+    logs: AppLog[]
+}
+
+export default class LogService {
+    public static async saveLog(
+        user: string | number,
```
Comment (obernardovieira) [inline packages/core/src/services/app/appLog.ts:16]: why not accept just id?

**Review by obernardovieira — CHANGES_REQUESTED** [review]

**Comment by dev-jotape** [comment]
@obernardovieira I made a refactor, could you validate again, please?

**Review by obernardovieira — CHANGES_REQUESTED** [review]

File: packages/core/src/services/app/user/log.ts [hunk 3]
```diff
@@ -0,0 +1,75 @@
...
+        }
+    }
+
+    public async get(ambassadorAddress: string, type: string, entity: string): Promise<AppLog[]> {
+        const community = await models.community.findOne({
+            attributes: ['id', 'contractAddress'],
+            where: {
+                ambassadorAddress,
+            }
+        });
```
[the remark this answers is not in the document]
Comment (dev-jotape) [reply packages/core/src/services/app/user/log.ts]: right! I changed a little the function

**Review by obernardovieira — APPROVED** [review]

**Pull request merged**